app/messages/_layout.email.html.erb

```diff
@@ -95,7 +95,9 @@ Margin is capitalized to fix Outlook.com
                 <%= content(:footer_link).html_safe %> &nbsp;|&nbsp;
               <% end %>
 
-              <a href="<%= communication_profile_url %>" style="white-space: nowrap;"><%= t 'click_to_preferences', "Update your notification settings" %></a>
+              <a href="<%= communication_profile_url %>" style="white-space: nowrap;">
+                <%= t '#messages.layout.email.html.click_to_preferences', "Update your notification settings" %>
```

Comment:
Hey @Gao-Jun You don't actually need the key here `'#messages.layout.email.html.click_to_preferences',` can be removed.  The translation function t now can just take a string.  If you want check out `app/views/discussion_topics/_group_discussion.html.erb` there is a translated string on line 33 that does exactly this.